angular-ngc/README.md

```diff
@@ -0,0 +1,25 @@
+# Angular
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.0.2 for an initial CLI repository.
+
+The lib-a and common projects added to demonstrate multiple projects with dependencies.
+
+Angular CLI logic was then removed and replaced with bazel.
```

Comment:
something needs to explain "what's the difference between angular and angular-ngc examples"

I think this README should explain what this one does, and link to the other folder explaining the trade-off.
Then the other README should do the same.

You can borrow some text from https://github.com/bazelbuild/rules_nodejs/blob/stable/docs/examples.md#angular

Reply:
I've updated all the docs. Could probably use some proof reading and review, let me know what's missing or you think can be improved etc.